### PR TITLE
Adds polkadot-notification-service

### PR DESCRIPTION
**Pull request by parutger**
Environment specifics to be placed in cloud-infra/kubernetes

### CODE REVIEW

File: charts/polkadot-notification-service/templates/deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+      serviceAccountName: {{ include "polkadot-notification-service.serviceAccountName" . }}
+      securityContext:
+        {{- toYaml .Values.podSecurityContext | nindent 8 }}
+      containers:
+        - name: {{ .Chart.Name }}
+          securityContext:
+            {{- toYaml .Values.securityContext | nindent 12 }}
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          env:
```
Comment (lazam) [inline charts/polkadot-notification-service/templates/deployment.yaml:34]: Let's include passing of additional env vars

**Comment by lazam** [comment]
Let's add service resource and service monitor resource

File: charts/polkadot-notification-service/templates/_helpers.tpl [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+Common labels
+*/}}
+{{- define "polkadot-notification-service.labels" -}}
+helm.sh/chart: {{ include "polkadot-notification-service.chart" . }}
+{{ include "polkadot-notification-service.selectorLabels" . }}
+{{- if .Chart.AppVersion }}
+app.kubernetes.io/version: {{ .Chart.AppVersion | quote }}
+{{- end }}
+app.kubernetes.io/managed-by: {{ .Release.Service }}
+{{- end }}
```
[the remark this answers is not in the document]
Comment (parutger) [reply charts/polkadot-notification-service/templates/_helpers.tpl:46]: Done!

**Comment by parutger** [comment]
> Let's add service resource
There is no external service for the service resource.
> and service monitor resource
I included the resource, but there is no metrics endpoint yet.

**Review by PierreBesson — APPROVED** [review]

**Pull request merged**